### ISSUE TITLE
Why the same seed is set but the training result is different?

### ISSUE THREAD

**Issue by Jmh0527** [open-mmlab/mmaction2#1596]
We used ipcsn for action recognition and set seed = 0, but the optimal accuracy on the validation set was different after each training. What could be causing this？

**Comment by kennymckormick** [comment]
Hi, you also need to set `torch.backends.cudnn.benchmark = True`. After setting this option, you can first try if you can get exactly the same results with single-GPU training. If  it's OK, then you can try multi-GPU (I'm not sure whether results for multi-gpu setting is exactly reproducible).

**Comment by Jmh0527** [comment]
> Hi, you also need to set `torch.backends.cudnn.benchmark = True`. After setting this option, you can first try if you can get exactly the same results with single-GPU training. If it's OK, then you can try multi-GPU (I'm not sure whether results for multi-gpu setting is exactly reproducible).

thanks for your timely help!
